client/components/utils/cursors-plugin.js

```diff
@@ -0,0 +1,86 @@
+import {Plugin} from 'prosemirror-state';
+import {Decoration, DecorationSet} from 'prosemirror-view';
+
+import {flatten} from './utils';
+/**
+ * Helper function that creates a DOM.node with the cursor to render
+ * @param {*} color - String with the hex color to use
+ */
+function createCaret(color, name, avatar) {
```

Comment:
Can you make this a Vue template?

Reply:
I'm not sure how to make a Vue template play nicely here because ProseMirror expects us to give it an actual DOM Node for it to render, and Vue manages its own Virtual DOM and renders to the actual DOM according to its lifecycle.

I asked in the forums for ideas: https://forum.vuejs.org/t/best-way-to-get-dom-node-from-template/27552?u=voluntadpear